src/KOKKOS/pair_pace_extrapolation_kokkos.h

```diff
@@ -0,0 +1,354 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef PAIR_CLASS
+// clang-format off
+PairStyle(pace/extrapolation/kk,PairPACEExtrapolationKokkos<LMPDeviceType>);
+PairStyle(pace/extrapolation/kk/device,PairPACEExtrapolationKokkos<LMPDeviceType>);
+PairStyle(pace/extrapolation/kk/host,PairPACEExtrapolationKokkos<LMPHostType>);
+// clang-format on
+#else
+
+// clang-format off
+#ifndef LMP_PAIR_PACE_EXTRAPOLATION_KOKKOS_H
+#define LMP_PAIR_PACE_EXTRAPOLATION_KOKKOS_H
+
+#include "pair_pace_extrapolation.h"
+#include "ace-evaluator/ace_radial.h"
```

Comment:
This include should be avoided. Any header declaring a "style" should not include any implementation specific headers.

Reply:
Upon closer inspection, this change **must** be done. The include file in question is in violation of the programming style requirement for LAMMPS. The standout issue is the use of "using namespace std;" in a header file. This effectively eliminates the benefits of namespaces and because all style files of the same kind are included at the same time in LAMMPS, will be implicitly applied to many other headers and classes and thus potentially cause namespace conflicts or unexpected access ambiguities beyond the style where it is applied. We've had very hard to debug problems of this kind before where adding a package would unexpectedly break some other packages.

In most of LAMMPS we go one step further and do not even use "using namespace std;" in the implementation to make it more obvious to people reading the code which are functions/classes of the C++ standard library and which are functions/members/instances of LAMMPS classes.